aten/src/ATen/native/BatchLinearAlgebra.cpp

```diff
@@ -237,6 +237,27 @@ TORCH_META_FUNC(triangular_solve)(const Tensor& self, const Tensor& A, bool uppe
   }
 }
 
+TORCH_META_FUNC(linalg_lu_factor_ex)(const Tensor& A, bool pivot, bool check_errors) {
+  TORCH_CHECK(A.dim() >= 2, "torch.lu_factor: Expected tensor with 2 or more dimensions. Got size: ", A.sizes(), " instead");
+
+  auto sizes = A.sizes().vec();
+  const auto m = sizes.cend()[-2];
+  const auto n = sizes.cend()[-1];
+
+  // make column major strides for BLAS
+  auto LU_strides = at::native::contiguous_strides(sizes, /*f-contig*=*/true);
```

Comment:
Just confirming- it sounds like `f-config` flag ends up doing the same thing as the as the deleted transpose code below?
```
    // Transpose size
    std::iter_swap(req_size.end() - 1, req_size.end() - 2);
    LU.resize_(req_size, MemoryFormat::Contiguous);
    LU.transpose_(-2, -1);  // make 'LU' have Fortran contiguous memory
```

Reply:
Yes, these two pieces of code are equivalent. This is just a convenient function to use in structured kernels.

Reply:
fwiw, this function was added in the previous PR of this stack: https://github.com/pytorch/pytorch/pull/67789#pullrequestreview-804938218